models/commit_status.go

```diff
@@ -261,7 +268,7 @@ func ParseCommitsWithStatus(oldCommits *list.List, repo *Repository) *list.List
 		commit := SignCommitWithStatuses{
 			SignCommit: &c,
 		}
-		statuses, err := GetLatestCommitStatus(repo, commit.ID.String(), 0)
+		statuses, err := GetLatestCommitStatus(repo.ID, commit.ID.String(), ListOptions{})
```

Comment:
So we load repo twice here?

Reply:
@lunny - no since GetLatestCommitStatus() only need repoID

see https://github.com/go-gitea/gitea/pull/14026/files#diff-4a9dd802dcff7a8403f71282d7238f1226f076e8989663501a4122fd2be4aa8cR149